public/styles/components/_form.scss

```diff
@@ -48,6 +48,12 @@
   padding-left: 0;
   padding-inline: 0;
   padding-block: 0;
+
+  &:first-child {
+    .bsds-form-control {
+      margin-top: space(2x);
```

Comment:
add
```
      & [class='bsds-fieldset'] {
        margin-top: space(4x);
      }
```

Reply:
This exact snippet wasn't working as expected, but I added a margin to fieldset within form on line 115